lctool/run.py

```diff
@@ -37,6 +37,44 @@ def lcget():
                 nf.write(content.encode('utf-8'))
                 nf.write('\n')
 
+def lcrefresh():
```

Comment:
I think it's a good idea. But I also come up another idea: add an parameter to lcget() function. But I am not familiar with python. Therefore don't know if this can be done? In another word, can I add parameter to an entry funcation? If this can be done, we can keep using lc-refresh command under the premise of simplifing code. (I still prefer using another command, as these two are somehow like git pull VS git fetch)



497018787@qq.com
 
From: zhtwd
Date: 2016-05-07 16:46
To: zhtwd/lctool
CC: Jiaxi He; Author
Subject: Re: [zhtwd/lctool] add command lc-refresh (#5)
In lctool/run.py:
> @@ -37,6 +37,44 @@ def lcget():
>                  nf.write(content.encode('utf-8'))
>                  nf.write('\n')
>  
> +def lcrefresh():

How about adding an argument such as '--override' to lc-get instead of making a new command since there are much overlap of the codes?
—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub